app/javascript/pages/map/recent-imagery/recent-imagery.js

```diff
@@ -19,6 +19,7 @@ import RecentImageryDrag from './recent-imagery-drag';
 import RecentImageryComponent from './recent-imagery-component';
 
 const LAYER_SLUG = 'sentinel_tiles';
+const MIN_ZOOM = 8;
```

Comment:
Again both of these could go in the reducer? Might make things easier later on if we move maps.